src/java.base/share/classes/java/lang/foreign/Linker.java

```diff
@@ -573,6 +591,22 @@ static Linker nativeLinker() {
      */
     SymbolLookup defaultLookup();
 
+    /**
+     * {@return a mapping between the names of data types used by the ABI implemented by this linker and their
```

Comment:
Much of the verbiage here is carried over from `defaultLookup` as we need to do the usual dance of saying that the set of returned types is not specified, but should be (a) sensible and (b) stable.